internal/dataplane/kong_client.go

```diff
@@ -575,12 +578,14 @@ func (c *KongClient) tryRecoveringWithFallbackConfiguration(
 
 	if failuresCount := len(fallbackParsingResult.TranslationFailures); failuresCount > 0 {
 		c.recordResourceFailureEvents(fallbackParsingResult.TranslationFailures, FallbackKongConfigurationTranslationFailedEventReason)
+		c.prometheusMetrics.RecordFallbackTranslationBrokenResources(failuresCount)
+		c.prometheusMetrics.RecordFallbackTranslationFailure()
 		c.logger.V(util.DebugLevel).Info("Translation failures occurred when building fallback data-plane configuration", "count", failuresCount)
+	} else {
+		c.prometheusMetrics.RecordFallbackTranslationBrokenResources(0)
+		c.prometheusMetrics.RecordFallbackTranslationSuccess()
 	}
 
-	// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/6082
-	// Expose Prometheus metrics for fallback configuration parsing result.
-
 	const isFallback = true
```

Comment:
Did I miss it or was this discussed somewhere already? Do we want this `const` here?

Reply:
Yeah, it was discussed [here](https://github.com/Kong/kubernetes-ingress-controller/pull/6099#discussion_r1618451178).